ibm/mas_devops/roles/mongodb/tasks/providers/community.yml

```diff
@@ -59,63 +59,43 @@
     apply: yes
     definition: "{{ lookup('template', 'templates/community/manager.yml') }}"
 
-
 # 5. Configure TLS using cert manager
 # -----------------------------------------------------------------------------
-#This task will create ClusterIssuer and wait till it gets ready
-- name: "Create a cluster issuer "
-  community.kubernetes.k8s:
+#Taks to create issuer in namespace which will be used to create other application related certificates
+- name: "Create a issuer  in {{ mongodb_namespace }}"
+  kubernetes.core.k8s:
     apply: yes
-    definition: "{{ lookup('template', 'templates/community/cluster-issuer.yml') }}"
+    definition: "{{ lookup('template', 'templates/community/issuer.yml') }}"
```

Comment:
This needs to be an issuer that is different to the issuer that signs the server certificate. At the moment you are using the same self-signed issuer for both so the server certificate will be self-signed rather than signed by the CA. The CA-issuer needs to be self-signed but the server-issuer needs to be signed by the results CA certificate created via the CA-issuer.

Reply:
done,Can you check the changes